app/connector/mongodb/src/main/resources/META-INF/syndesis/connector/mongodb3.json

```diff
@@ -156,10 +184,9 @@
           "io.syndesis.connector.mongo.MongoProducerUpdateCustomizer"
         ],
         "inputDataShape": {
-          "name": "Criteria",
-          "description": "Text criteria",
-          "kind": "java",
-          "type": "java.lang.String"
+          "name": "Filter parameters",
+          "description": "Flat list of parameters contained in the filter provided by user",
```

Comment:
This will be presented in the user interface; I don't think we need to tell the user that this is _provided by user_. Perhaps discuss with @TovaCohen for a better way to phrase these.

Reply:
Yes, this is something that will be reviewed and adjusted with the feedback received by doc and product team.